plugins/catalog/api-report.md

```diff
@@ -371,7 +371,9 @@ export interface HasSystemsCardProps {
 }
 
 // @public
-export function isComponentType(type: string): (entity: Entity) => boolean;
+export function isComponentType(
+  types: string | string[],
```

Comment:
Do you want to add this to `isKind` and `isNamespace` too? :grin: Feel that would make sense